app/Result.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Result extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'total', 'user_id', 'test_id',
```

Comment:
In migrations, you name the column "total_score", but there you've used "total" for the column name.
I think the name "score" will be better.